module/VuFind/src/VuFind/ILS/Driver/PAIA.php

```diff
@@ -239,7 +270,7 @@ public function cancelHolds($cancelDetails)
             $array_response = $this->paiaPostAsArray(
                 'core/' . $patron['cat_username'] . '/cancel', $post_data
             );
-        } catch (ILSException $e) {
+        } catch (Exception $e) {
```

Comment:
Shouldn't this be `catch (\Exception $e) {` -- otherwise it's going to be looking for a non-existent `VuFind\ILS\Driver\Exception` class.

Reply:
Thanks for the fix!